build.js

```diff
@@ -35,18 +36,32 @@ var isOptional = function isOptional(category) {
   return (category || '').indexOf('annex b')>-1 || category === 'pre-strawman' || category === 'strawman (stage 0)';
 };
 
+var byTestSuite = function(suite) {
+  return function(browser) {
+    return ('test_suites' in browser) ? browser.test_suites.indexOf(suite)>-1 : true;
```

Comment:
instead of just checking `in`, could we use https://npmjs.com/has, and/or check that `Array.isArray`?